tgarchive/build.py

```diff
@@ -178,7 +183,22 @@ def _make_abstract(self, m, media_mime):
     def _nl2br(self, s) -> str:
         # There has to be a \n before <br> so as to not break
         # Jinja's automatic hyperlinking of URLs.
-        return _NL2BR.sub("\n\n", s).replace("\n", "\n<br />")
+        return _NL2BR.sub("\n\n", str(s)).replace("\n", "\n<br />")
```

Comment:
Can't we use this? Whether the s variable is a string or not?

Reply:
Yes, it can be removed now as [urlize()](https://github.com/faraazb/tg-archive/blob/24abc2bf5c10fb905629d7e40c8a12eebe59df04/tgarchive/build.py#L195) does a cast already.
The re.sub() method expects a string and there are NULL/None messages which need to be cast to string. This was indirectly taken care of by the escape filter in the master branch. Since I changed the way filters work in the first commit, I had to make this cast.